catroid/src/main/java/org/catrobat/catroid/cloudmessaging/CloudMessaging.java

```diff
@@ -54,6 +61,30 @@ public void showNotification(CloudMessage cloudMessage) {
 		}
 	}
```

Comment:
Please move the `showNotification` to the `buildNotification` method. Things that belongs together shouldn't be separated.
I consider to split the show/build notification and the initialization into two classes. However, for now it's fine to me.